docs/developers-guide/debug.md

```diff
@@ -80,6 +80,11 @@ rodny@deepblue-vm-1:~$ cd ~/go/src/github.com/nestybox/sysbox/sysbox-runc
 
 -   Attaching to a running process:
 
+First, find the PID of the running sysbox-runc process. Use `sudo pstree -SlpgT`
+to help with this.
```

Comment:
If we go to this level of detail, i would then give something more specific (e.g. pstree -SlpgT | grep sysbox).

Reply:
Done!